pageserver/src/walingest.rs

```diff
@@ -151,246 +152,147 @@ impl WalIngest {
     /// This function returns `true` if the record was ingested, and `false` if it was filtered out
     pub async fn ingest_record(
```

Comment:
nit: update the doc comment to reflect that it no longer decodes records.

Reply:
done in https://github.com/neondatabase/neon/pull/9524/commits/f8f5c4e18aefa6f599c1ea343458ee667495e855